src/db/clickhouse-migrations.test.ts

```
@@ -12,12 +12,61 @@ vi.mock("pg", () => ({
   escapeIdentifier: (value: string) => `"${value.replaceAll('"', '""')}"`,
 }));
 
+import { createMigration as createBackfillMaterializedMetricStreamMigration } from "./clickhouse-migrations/0005_backfill_materialized_metric_stream.ts";
```

Biome (2.4.15)

[error] Type assertion with 'as' is banned. Use Zod parsing, type guards, generics, or 'satisfies' instead.

(plugin)